packages/stream/src/combinators/state.ts

```diff
@@ -0,0 +1,53 @@
+import { Stream } from '@motorcycle/types'
+import { curry3 } from '167'
+import { scan } from './scan'
+import { switchMap } from './switchMap'
+
+/**
+ * Especially useful when keeping local state that also needs to be updated
+ * from an outside source.
```

Comment:
Correction:
> from a source.

Note: a source _always_ comes from the other side of the boundary.